doc/releases/release-notes-2.5.rst

```diff
@@ -303,6 +303,7 @@ MCUBoot
     see ``CONFIG_MCUBOOT_CLEANUP_ARM_CORE``.
   * Allow the final data chunk in the image to be unaligned in
     the serial-recovery protocol.
+  * Added `revert` support to the direct-xip mode.
```

Comment:
Literal in rST uses two back quotes around the text.

Reply:
fixed